main.go

```diff
@@ -179,7 +182,14 @@ func sendToScope(testSuite *JUnitTestSuite) {
 			span.SetTag("test.status", tags.TestStatus_PASS)
 		}
 
-		span.FinishWithOptions(opentracing.FinishOptions{FinishTime: time.Now()})
+		durationInSecs, err := strconv.ParseFloat(testCase.Time, 64)
+		if err != nil {
+			log.Println(testCase.Time + " cannot be parsed as number")
+			durationInSecs = 0
+		}
+
+		timeEndSpan := timeStartSpan.Add(time.Duration(durationInSecs * 1_000_000_000)) //to nanoseconds
```

Comment:
If you are adding a `time.Duration`, you cannot pass nanoseconds right?
```suggestion
		timeEndSpan := timeStartSpan.Add(time.Duration(durationInSecs * time.Second)) // to nanoseconds
```

Reply:
Changed this logic to use the function `time.ParseFunction(string)`.